### PR TITLE
Add partially support for MS-7751 boards (F71889AD)

### PR DESCRIPTION
**Pull request by wlkmanist**
image

### CODE REVIEW

**Comment by cristianst85** [comment]
I have an MSI Z77A-GD65 board and gave this PR a try. Mine also shows the GPU voltage and CPU Fan Speed. I don't have any other controllable fans connected to the mainboard.

image

**Comment by wlkmanist** [comment]
> I have an MSI Z77A-GD65 board and gave this PR a try. Mine also shows the GPU voltage and CPU Fan speed. I don't have any other controllable fans connected to the mainboard.
> 
> image

It should work fine for your motherboard, but for MPower it is not a full support, because of one more hw monitor chip.
On my pc configuration gpu is disabled, and I have 2 more fans (5 in total).

Also you can check my fork (master branch). I added more MSI boards (Z77, Z68, X79), full MPower support.

File: LibreHardwareMonitorLib/Hardware/Motherboard/Identification.cs [hunk 1]
```diff
@@ -273,6 +273,15 @@ public static Model GetModel(string name)
...
                 return Model.Z270_PC_MATE;
+            case var _ when name.Equals("Z77 MPower", StringComparison.OrdinalIgnoreCase):       // MS-7751 Rev 4.x
+            case var _ when name.Equals("Z77 MPower (MS-7751)", StringComparison.OrdinalIgnoreCase):
+            case var _ when name.Equals("Z77A-GD65", StringComparison.OrdinalIgnoreCase):        // MS-7751 Rev >1.x
+            case var _ when name.Equals("Z77A-GD65 (MS-7751)", StringComparison.OrdinalIgnoreCase):
+            case var _ when name.Equals("Z77A-GD65 GAMING", StringComparison.OrdinalIgnoreCase): // MS-7751 Rev 2.x
+            case var _ when name.Equals("Z77A-GD65 GAMING (MS-7751)", StringComparison.OrdinalIgnoreCase):
+            case var _ when name.Equals("Z77A-GD55", StringComparison.OrdinalIgnoreCase):        // MS-7751 Rev 1.x
+            case var _ when name.Equals("Z77A-GD55 (MS-7751)", StringComparison.OrdinalIgnoreCase):
+                return Model.Z77_MPower;
```
Comment (cristianst85) [inline LibreHardwareMonitorLib/Hardware/Motherboard/Identification.cs:284]: If I may, why not use something more generic like `Model.Z77_MS7751` instead?

Comment (wlkmanist) [reply LibreHardwareMonitorLib/Hardware/Motherboard/Identification.cs:284]: Its "legacy". Platform is the same, some Z68 boards, that are not an MS-7751 also have the same logic. I think that variable name is not so important.

Comment (wlkmanist) [reply LibreHardwareMonitorLib/Hardware/Motherboard/Identification.cs:284]: Also the same code works for the "Z68A-GD80 (MS-7757)" and similiar MSI motherboards with non Intel XE chipset and with F71889AD. This code I will PR later, and it is available in my fork (master).

**Pull request merged**